packages/cli/README.md

```diff
@@ -273,6 +273,122 @@ bubblewrap fingerprint generateAssetLinks <flags>
+|features|[Features](#Features)|false|Enables optional features in the Android application. Read the [Features](#Features) section for details.|
+|alphaDependencies|[AlphaDependencies](#AlphaDependencies)|false|Enables the Android application to use alpha version of dependencies. Defaults to `false`.|
+|enableSiteSettingsShortcut|boolean|false|Adds a shortcut to the site settins in the application launcher. Defaults to `true`.|
+|isChromeOSOnly|boolean|false|Generates an application that targets only ChromeOS devices. Defaults to `false`.|
+|shareTarget|[ShareTarget](https://w3c.github.io/web-share-target/#sharetarget-and-its-members)|false||
+|orientation|`'default'` \| `'any'` \| `'natural'` \| `'landscape'` \| `'portrait'` \| `'portrait-primary'` \| `'portrait-secondary'` \| `'landscape-primary'` \| `'landscape-secondary'`|false|Initial orientation used to launch the Android application. Defaults to `'default'`.|
+|fingerprints|[Fingerprint](#fingerprint)[]|false|List of fingerprints used to generate the Digital Asset Links file. Read the [Fingerprint](#fingerprint) section for details.|
+|serviceAccountJsonFile|string|false|The Play Store serviced account information. Currently unused.|
+|additionalTrustedOrigins|string[]|false|A list of additional origins owned by developer and validated with Digital Asset Links. The user will remain in fullscreen mode when navigating to those origins inside the application.|
+ 
+### Features
+
+Developers can enable additional features in their Android application. Some features may include more dependencies into the application and increase the birary size.
+
+|Name|Type|Required|Description|
+|:--:|:--:|:------:|:---------:|
+|locationDelegation|[LocationDelegationConfig](#locationdelegationconfig)|false|Read the [LocationDelegationConfig](#locationdelegationconfig) section for details.|
+|playBilling|[PlayBillingConfig](#playbillingconfig)|false|Read the [PlayBillingConfig](#playbillingconfig) section for details.|
+|firstRunFlag|[FirstRunFlagConfig](#firstrunflagconfig)|false|Read the [FirstRunFlagConfig](#firstrunflagconfig) section for details.|
+|appsFlyer|[AppsFlyerConfig](#appsflyerconfig)|false|Read the [AppsFlyerConfig](#appsflyerconfig) section for details.|
+
+#### AppsFlyerConfig
+Enables the [AppsFlyer SDK](https://support.appsflyer.com/hc/en-us/articles/360002330178-Using-AppsFlyer-with-TWA#introduction) in the Android application. Includes additional libraries and is not compatible with Chrome OS.
+
+|Name|Type|Required|Description|
+|:--:|:--:|:------:|:---------:|
+|enabled|boolean|true|Set to `true` to enable the feature.|
+|appsFlyerId|boolean|true|The appsflyer id.|
+
+#### LocationDelegationConfig
+
+Delegates the location permission dialog to the Android system, instead of showing the browser dialog. Recommended when the application requests the location permission, but not enabled by default as it includes additional dependencies in the application. 
```

Comment:
Mention the binary size increase?

Reply:
Or maybe the Features table could have a binary size increase column for each feature.

Reply:
Added a column.